tests/models/test_units.py

```diff
@@ -64,7 +64,7 @@ def test_unit_model_caching(self):
 
         self.assertEqual(unit.__hash__(), unit.__hash__())
 
-        @functools.lru_cache
+        @functools.lru_cache(maxsize=128)
```

Comment:
isn't that the default anyway? I don't think that python 3.7 should fail because of this. https://docs.python.org/3.7/library/functools.html

Reply:
I also wonder about that, but it actually fails with the error message:
```
TypeError: Expected maxsize to be an integer or None
```
btw, the python examples all give the `maxsize` explicitly.